apps/wolfsshd/wolfsshd.c

```diff
@@ -2374,8 +2374,8 @@ static int StartSSHD(int argc, char** argv)
 #ifdef WOLFSSL_NUCLEUS
             struct addr_struct clientAddr;
 #else
-            SOCKADDR_IN_T clientAddr;
-            socklen_t     clientAddrSz = sizeof(clientAddr);
+            struct sockaddr_in6 clientAddr;
```

Comment:
Is there a reason we don't have all this wrapped with `TEST_IPV6` and use `SOCKADDR_IN_T` in the non IPv6 case? I noticed a bit of a mess between the examples for the IPv4 and IPv6 support.

Reply:
Already discussed internally.  No need for changes.  Back to @dgarske.

Reply:
The ssd will always support IPv6